server/planning/feeding_services/onclusive_api_service.py

```diff
@@ -101,18 +101,13 @@ def _update(self, provider, update):
                 logger.info("Ingesting content: {} ...".format(str(between_event_response.content)[:4000]))
 
                 if hasattr(parser, "parse_http"):
```

Comment:
are actually both options possible? imo we should limit onclusive to single parser as we do for rss

Reply:
as much as i understand, that we have to registered a feedparser in the feeding service just like rss or we should do something like:
```
 if hasattr(parser, "parse_onclusive"):
      yield (parser.parse_onclusive(content, provider))
```

Reply:
we can set the parser explicitly like here:
https://github.com/superdesk/superdesk-core/blob/1b3ff89a92d3cce8e1cbfe635f4c23c6e39f9a5c/superdesk/io/feeding_services/rss.py#L403
so then we know it will be using onclusive parser and the parser will have that `parse_http` method so that if is not needed anymore